packages/service-library/src/servicelib/fastapi/long_running_tasks/_client.py

```diff
@@ -122,6 +122,10 @@ def __init__(self, app: FastAPI, async_client: AsyncClient, base_url: str):
         self._async_client = async_client
         self.base_url = base_url
 
+    @property
+    def external_reference(self) -> str:
```

Comment:
minor: not sure I find this very explicit or even better than `base_url`. you use that only for a tip.
I guess this will be removed?